### DISCUSSION TITLE
TX Disable excluding NZ PRS Bands

### DISCUSSION THREAD

**Discussion by 1randomkiwi** [egzumer/uv-k5-firmware-custom#488] [Ideas]
It would be great to have a TX Disable option that blocked TX except for the New Zealand CB (26 MHz and 27 MHz) and the PRS (477 MHz) bands. (https://rrf.rsm.govt.nz/ui/licence/radio/view/146140)
Even better would be for the ability to enter custom bands to enable TX

**Comment by egzumer** [comment]
You really, really shouldn't be transmitting on 26 or 27MHz with this radio (or anywhere outside 70cm or 2m band for that matter).
I don't plan on adding any additional tx locks, this is a HAM transceiver and a user is responsible for not transmitting where he shouldn't.

You can disable TX for each memory channel separately by setting an offset to 0MHz, or by setting duplex to off in CHIRP (it does the same thing)

**Comment by 1randomkiwi** [comment]
Thanks for the quick reply and insight. I was hoping to have a quick and easy way of locking things down, but the channel locking and disabling frequency mode should do the trick.

And agree, I don't plan to ever transmit on the 26-27MHz band with this, just thought I would align my request with the GURL.

And thanks for all the work you've put into this firmware. It makes a great little transceiver and even better one!
